### PR TITLE
JB-736 - Pay field 'Receive X tokens per ETH'

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Closes [JB-736 : Pay field 'Receive X tokens per ETH'](https://linear.app/peel/issue/JB-736/pay-field-receive-x-tokens-per-eth)

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #4021** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4021" target="_blank">Graphite</a> 
    * **PR #4022** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4022" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4022?utm_source=stack-comment).

File: src/components/ProjectDashboard/components/PayProjectCard/components/__snapshots__/TokensPerEth.test.tsx.snap [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 <div>
-  Receive 100 TKN
+  Receive 100 TKN 
 </div>
 `;
 
 exports[`TokensPerEth renders when \`currencyAmount\` is undefined 1`] = `
 <div>
-  Receive 100 TKN/1 ETH
+  Receive 100 TKN [object Object]
```
Comment (tomquirk) [inline src/components/ProjectDashboard/components/PayProjectCard/components/__snapshots__/TokensPerEth.test.tsx.snap:11]: eh O.o

**Review by tomquirk — APPROVED** [review]

File: src/components/ProjectDashboard/components/PayProjectCard/components/__snapshots__/TokensPerEth.test.tsx.snap (hunk 1 above)
Comment (wraeth-eth) [reply src/components/ProjectDashboard/components/PayProjectCard/components/__snapshots__/TokensPerEth.test.tsx.snap:11]: yeah personally starting to hate snapshots lol

**Comment by wraeth-eth** [comment]
@wraeth-eth started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4022).

**Comment by wraeth-eth** [comment]
[Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4022) rebased this pull request as part of a merge.

**Comment by wraeth-eth** [comment]
@wraeth-eth merged this pull request with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4022).

**Pull request merged**